ddl/disttask_flow.go

```diff
@@ -0,0 +1,127 @@
+// Copyright 2023 PingCAP, Inc.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//	http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package ddl
+
+import (
+	"context"
+	"encoding/json"
+
+	"github.com/pingcap/errors"
+	"github.com/pingcap/tidb/disttask/framework/dispatcher"
+	"github.com/pingcap/tidb/disttask/framework/proto"
+	"github.com/pingcap/tidb/kv"
+	"github.com/pingcap/tidb/meta"
+	"github.com/pingcap/tidb/parser/model"
+)
+
+// FlowHandleLitBackfillType is the task type to handle backfill.
+const FlowHandleLitBackfillType = "flowHandleLitBackfill"
+
+// LitBackfillGlobalTaskMeta is the global task meta for lightning backfill.
+type LitBackfillGlobalTaskMeta struct {
+	Job          model.Job `json:"job"`
+	EleID        int64     `json:"ele_id"`
+	EleTypeKey   []byte    `json:"ele_type_key"`
+	IsMergingIdx bool      `json:"is_merging_idx"`
```

Comment:
IsMergingIdx is not needed, we can know it from the step.

Reply:
User can only set the step when global task is created: https://github.com/pingcap/tidb/blob/3f8b0ccd81374a27a466db2e934737503905b30c/disttask/framework/storage/task_table.go#L131-L138

And I think it's reasonable, all tasks should start from a `init` step. So we need user to put some additional informations into meta to separate different global tasks.

Reply:
If so, the tp should be different?

Reply:
I have no strong opinion for whether introduce a new type of global task or put some variable into the meta. However will update the PR to make two step different types which sound also great too.